ComposeSample/app/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
@CesarValiente And how about this file? Not sure about the rules of the system-created files.

Reply:
Good catch! you can get rid of this file since we don't plan to publish (and then shrink and obfuscate) the app.
When removing it, please, get rid as well in the `ComposeSample/app/build.gradle` file the node:
```
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
        }
    }
```
That references to it configuring release build type.